sdk/tests/conformance/textures/00_test_list.txt

```diff
@@ -1,5 +1,6 @@
 misc/00_test_list.txt
 canvas/00_test_list.txt
+--min-version 1.0.4 canvas_sub_rectangle/00_test_list.txt
```

Comment:
I think 1.0.4 is the conceptually correct version, since these tests live in the "WebGL 1.0" sub-part of the tests' tree (conformance/ rather than conformance2/) -- this is because the texture formats that are tested in this directory are those which WebGL 1.0 supported.

Right now, the tests all skip themselves if they're run on WebGL 1.0 -- the expectation is that we *might* add this sub-source functionality to WebGL 1.0 later.